automation_tools/__init__.py

```diff
@@ -1144,8 +1144,9 @@ def product_install(distribution, create_vm=False, certificate_url=None,
                 execute(setup_foreman_discovery, host=host)
             if os.environ.get('LIBVIRT_KEY_URL') is not None:
                 execute(setup_libvirt_key, host=host)
-            execute(install_puppet_scap_client, host=host)
-            execute(setup_oscap, host=host)
+            if os.environ.get('SATELLITE_VERSION') == '6.1':
```

Comment:
Let's do that when we add support for Sat 6.2 and not now. Currently SATELLITE_VERSION just supports version 6.0 and 6.1